src/components/Application.pcss

```diff
@@ -67,23 +67,22 @@ code {
 
 a {
     text-decoration: none;
-    color: var(--blue-accent);
+    color: var(--main-color-text);
```

Comment:
Application link colors should be `--blue-accent`. Please, change colors for components, not globally.